vimish-fold.el

```diff
@@ -224,6 +233,21 @@ This includes fringe bitmaps and faces."
         '(vimish-fold--folded
           vimish-fold--unfolded)))
 
+(defun vimish-fold--vimish-overlay-at (pos)
+  (let ((overlays (overlays-at pos)) res-ov tmp-ov (min-space (max-char)))
```

Comment:
Emacs says:

> All variables and subroutines might as well have a documentation string

I understand that this is nitpicking, but since we're already following the whims of Emacs Lisp official style guide, there  is no reason to stop following them now :-)

Reply:
Done.

I'm not used enough to elisp dev I guess ^^"